akka-typed/src/main/scala/akka/typed/EventStream.scala

```diff
@@ -66,14 +68,15 @@ object Logger {
 }
 
 class DefaultLogger extends Logger with StdOutLogger {
-  import ScalaDSL._
   import Logger._
 
-  val initialBehavior =
-    ContextAware[Command] { ctx ⇒
-      Total {
-        case Initialize(eventStream, replyTo) ⇒
-          val log = ctx.spawn(Deferred[AnyRef] { () ⇒
+  val initialBehavior = {
+    // TODO avoid depending on dsl here?
```

Comment:
core packages depending on concrete stuff feels backwards, but I'm not sure we can do it in any other way.